README.md

```diff
@@ -143,7 +143,7 @@ Windows using Visual Studio
 All needed library dependencies are provided with the Simbody installation on Windows, including linear algebra and visualization dependencies. 
 
 1. Download and install [Microsoft Visual Studio](http://www.visualstudio.com), version 2013 or higher. The "Community Edition" is free for "non-enterprise" use. The "Express" edition is another free option, in that case use *Visual Studio Express for Windows Desktop*.
-2. Download and install [CMake](http://www.cmake.org/download), version 2.8.6 or higher.
+2. Download and install [CMake](http://www.cmake.org/download), version 2.8.8 or higher.
```

Comment:
For Ubuntu 12.04, you can recommend a PPA, which makes it easier to install custom packages. [Here's one](https://launchpad.net/~robotology/+archive/ubuntu/ppa) for example.